### PR TITLE
Release notes for release 2.2.0

### PR DESCRIPTION
**Pull request by aindriu-aiven**
Release notes highlighting new 2.2.0 release and some of the features available in this release.

### CODE REVIEW

File: docs/releases/release220.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Pre Validation of Schema Compatibility
+``````````````````````````````````````
+Klaw will now check all Schema compatibility against the Schema Registry before creating the Schema Creation requests. This feature will only be bypassed used when forceRegister is in use.
+
+ACL information is now masked
+`````````````````````````````
+Klaw now masks IP Address's and Service Account names from all users except the ACL owner, ensuring better security for users.
```
Comment (muralibasani) [inline docs/releases/release220.rst:39]:
```suggestion
Klaw now masks IP Addresses, certificates and Service Account credentials from all users except the ACL owner, ensuring better security for users.
```

File: docs/releases/release220.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+::
+    # Enable new Klaw user interface
+    klaw.coral.enabled=true
+
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Pre Validation of Schema Compatibility
+``````````````````````````````````````
+Klaw will now check all Schema compatibility against the Schema Registry before creating the Schema Creation requests. This feature will only be bypassed used when forceRegister is in use.
```
Comment (muralibasani) [inline docs/releases/release220.rst:35]: It's abit unclear here 'bypassed used when forceRegister'

File: docs/releases/release220.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
+Release 2.2.0
+=============
+
+Date: 3rd of April, 2023
```
Comment (muralibasani) [inline docs/releases/release220.rst:4]:
```suggestion
Date: 5th of April, 2023
```

**Review by muralibasani — APPROVED** [review]
LGTM, thanks

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
Looks good. A few minor suggestions.

File: docs/releases/release220.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+`````````````````````````````````````````
+With the switch to React, this release features a redesign of the following user interfaces:
+
+- **My Team's Topic Requests**: Users can now view all of their team's topic requests using the new UI.
+
+- **My Team's Subscription Requests** (Producer/Consumer):  Users can now view all of their team's ACL requests for both producers and consumers using the new UI.
+
+- **My Team's Schema Requests**:  Users can now view all of their team's schema requests using the new UI.
+
+.. image:: /../../../_static/images/ApprovalRequestTopic-react.png
```
Comment (programmiri) [inline docs/releases/release220.rst:25]: wrong  🖼️  link!

**Review by programmiri — APPROVED** [review]

File: docs/releases/release220.rst [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+To preview the new Klaw user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
+::
+    # Enable new Klaw user interface
+    klaw.coral.enabled=true
+
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Pre Validation of Schema Compatibility
```
Comment (harshini-rangaswamy) [inline docs/releases/release220.rst:34]:
```suggestion
Pre-validation of schema compatibility
```

File: docs/releases/release220.rst [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+``````````````````````````````````````
+Klaw will now check all Schema compatibility against the Schema Registry before creating the Schema Creation requests. This feature will only be bypassed during promotion when the 'force register schema' option is selected.
```
Comment (harshini-rangaswamy) [inline docs/releases/release220.rst:36]:
```suggestion
Klaw will now check all schema compatibility against the Schema Registry before creating the schema creation requests. This feature will only be bypassed during promotion when the 'force register schema' option is selected.
```

File: docs/releases/release220.rst [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+ACL information is now masked
```
Comment (harshini-rangaswamy) [inline docs/releases/release220.rst:38]:
```suggestion
Masking of ACL information for enhanced security
```

File: docs/releases/release220.rst [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+`````````````````````````````
+To enhance security, Klaw now masks IP Addresses, certificates, and Service Account credentials from all users except the ACL owner team. This enhancement ensures that sensitive information is only visible to authorized personnel, improving overall security.
+
+Export Data to JSON
```
Comment (harshini-rangaswamy) [inline docs/releases/release220.rst:42]:
```suggestion
Export data to JSON
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
Changes to sentence casing.

**Review by harshini-rangaswamy — APPROVED** [review]

**Pull request merged**